tasks/deploy.yaml

```diff
@@ -98,9 +98,9 @@ spec:
 
         echo "deploying to ephemeral"
         if [ -z "${EXTRA_DEPLOY_ARGS}" ]; then
-          EXTRA_DEPLOY_ARGS="$(parse-snapshot.py)"
+          EXTRA_DEPLOY_ARGS=" $(parse-snapshot.py) "
         else
-          EXTRA_DEPLOY_ARGS+="$(parse-snapshot.py)"
+          EXTRA_DEPLOY_ARGS+="$ (parse-snapshot.py) "
```

Comment:
whitespace in the wrong spot, otherwise I think this is a sane change. Maybe there is something to be done in cicd-tools-konflux deploy.sh to process these args without this whitespace being lost.